### DISCUSSION TITLE
WG Governance Framework v4.0.0

### DISCUSSION THREAD

**Discussion by lvnacy** [BanklessDAO/writers-guild-governance-framework#3] [Announcements]
Writers Guild Governance Framework port from Google Docs to GitHub with updated processes from proposals that were ratified during BanklessDAO’s Seasons 5 & 6.

<hr /><em>This discussion was created from the release <a href='https://github.com/BanklessDAO/writers-guild-governance-framework/releases/tag/4.0.0'>WG Governance Framework v4.0.0</a>.</em>

**Comment by sianliu** [comment]
I don't see anything. Is it because I'm on mobile?

**Reply by lvnacy** [reply]
This is just a placeholder for the Framework where we can hold a discussion around it. You can find the framework here:
[Governance Framework](https://github.com/BanklessDAO/writers-guild-governance-framework/blob/main/Policy%20docs/Writers%20Guild%20Governance%20Framework.md)
